azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/components/JoinCallButtonHolderView.kt

```diff
@@ -47,8 +49,13 @@ internal class JoinCallButtonHolderView : ConstraintLayout {
         viewModel: JoinCallButtonHolderViewModel
     ) {
         this.viewModel = viewModel
-        setupJoinCallButtonText.text = context.getString(R.string.azure_communication_ui_calling_setup_view_button_join_call)
-        joiningCallText.text = context.getString(R.string.azure_communication_ui_calling_setup_view_button_connecting_call)
+        if (viewModel.getCallType() == CallType.ONE_TO_N_CALL_OUTGOING) {
```

Comment:
Opinion here is that CallType should be hidden from view (Business logic).

VM should feed up the Resource ID's, button visibility, etc. Things directly relevant to display/view.

Reply:
As per discussion we will fix for GA.